static/usage/list-header/lines/react.md

```diff
@@ -4,26 +4,43 @@ import { IonItem, IonLabel, IonList, IonListHeader } from '@ionic/react';
 
 function Example() {
   return (
-    <IonList>
-      <IonListHeader>
-        <IonLabel>Video Games</IonLabel>
-      </IonListHeader>
-      <IonItem>
-        <IonLabel>Pokémon Yellow</IonLabel>
-      </IonItem>
-      <IonItem>
-        <IonLabel>Mega Man X</IonLabel>
-      </IonItem>
-      <IonItem>
-        <IonLabel>The Legend of Zelda</IonLabel>
-      </IonItem>
-      <IonItem>
-        <IonLabel>Pac-Man</IonLabel>
-      </IonItem>
-      <IonItem>
-        <IonLabel>Super Mario World</IonLabel>
-      </IonItem>
-    </IonList>
+    <>
+      <IonList>
+        <IonListHeader>
+          <IonLabel>Default</IonLabel>
+        </IonListHeader>
+        <IonItem>
+          <IonLabel>Item</IonLabel>
+        </IonItem>
+        <ion-item>
```

Comment:
This needs to be converted to react format:

```jsx
<IonItem>
    <IonLabel>Item</IonLabel>
</IonItem>
```

Reply:
Good catch 👍